package.js

```diff
@@ -1,48 +1,54 @@
+/* eslint-disable no-undef */
+
 Package.describe({
-  name: "mizzao:user-status",
-  summary: "User connection and idle state tracking for Meteor",
-  version: "0.6.8",
-  git: "https://github.com/mizzao/meteor-user-status.git"
+  name: 'mizzao:user-status',
+  summary: 'User connection and idle state tracking for Meteor',
+  version: '0.7.0',
+  git: 'https://github.com/mizzao/meteor-user-status.git'
 });
 
-Package.onUse( function(api) {
-  api.versionsFrom("1.2.0.1");
+Package.onUse((api) => {
+  api.versionsFrom('1.2.0.1');
 
+  api.use('ecmascript');
   api.use('accounts-base');
   api.use('check');
-  api.use(['coffeescript@1.0.9||2.0.0', 'underscore']);
+  api.use('underscore');
   api.use('mongo');
+  api.use('tracker', 'client');
+  api.use('mizzao:timesync@0.5.1');
 
-  api.use('deps', 'client');
-
-  api.use('mizzao:timesync@0.3.4');
-
-  api.addFiles('monitor.coffee', 'client');
-  api.addFiles('status.coffee', 'server');
+  api.export('MonitorInternals', 'client', {
+    testOnly: true
+  });
+  api.export('StatusInternals', 'server', {
+    testOnly: true
+  });
 
-  api.export('UserStatus'); // on both
+  api.mainModule('client/monitor.js', 'client');
+  api.mainModule('server/status.js', 'server');
```

Comment:
Using `mainModule` came to Meteor in [1.7](https://github.com/meteor/meteor/blob/devel/History.md#v17-2018-05-28), so I suggest we have our minimum version either to that or better 1.7.0.5.